test/entropic/sinkhorn.jl

```diff
@@ -195,4 +196,93 @@ Random.seed!(100)
             @test μ_interp ≈ μ_interp_pot rtol = 1e-6
         end
     end
+
+    @testset "sinkhorn divergence" begin
+        @testset "univariate exmaples" begin
+            # create distributions 
+            n = 20
+            m = 10
+            μsupp = [rand(1) for i in 1:n]
+            νsupp = [rand(1) for i in 1:m]
+            μprobs = normalize!(rand(n), 1)
+            μ = OptimalTransport.discretemeasure(μsupp, μprobs)
+            ν = OptimalTransport.discretemeasure(νsupp)
+
+            for (ε, metrics) in Iterators.product(
+                [0.1, 1.0, 10.0],
+                [
+                    (sqeuclidean, SqEuclidean()),
+                    (euclidean, Euclidean()),
+                    (totalvariation, TotalVariation()),
+                ],
+            )
```

Comment:
This is a bit uncommon, usually one would just use two `for` loops here. One can even use the short notation
```julia
for x in xs, y in ys
```